plotter.go

```diff
@@ -71,7 +71,7 @@ func Plot(expr PlotExpr, queryTime time.Time, duration, resolution time.Duration
 		resolution,
 	)
 	if err != nil {
-		_ = bugsnag.Notify(errors.Wrap(err, "error querying Prometheus"), nil,
+		_ = bugsnag.Notify(err, nil,
```

Comment:
I dont think you need to pass the `nil` here, you can just pass `bugsnag.Metadata` as the second argument